test/test_interpolations.jl

```diff
@@ -47,19 +47,29 @@
         # Note that not every element formulation exists for every order and dimension.
         applicable(Ferrite.getlowerorder, interpolation) && @test typeof(Ferrite.getlowerorder(interpolation)) <: Interpolation{ref_shape,func_order-1}
 
-    # Check partition of unity at random point.
-    n_basefuncs = getnbasefunctions(interpolation)
-    x = rand(Tensor{1, ref_dim})
-    f = (x) -> [shape_value(interpolation, Tensor{1, ref_dim}(x), i) for i in 1:n_basefuncs]
-    @test vec(ForwardDiff.jacobian(f, Array(x))') ≈
-        reinterpret(Float64, [shape_gradient(interpolation, x, i) for i in 1:n_basefuncs])
-    @test sum([shape_value(interpolation, x, i) for i in 1:n_basefuncs]) ≈ 1.0
-
-    # Check if the important functions are consistent
-    coords = Ferrite.reference_coordinates(interpolation)
-    @test length(coords) == n_basefuncs
-    @test shape_value(interpolation, x, n_basefuncs) == shape_value(interpolation, x, n_basefuncs)
-    @test_throws ArgumentError shape_value(interpolation, x, n_basefuncs+1)
+        n_basefuncs = getnbasefunctions(interpolation)
+        coords = Ferrite.reference_coordinates(interpolation)
+        f(x) = [shape_value(interpolation, Tensor{1, ref_dim}(x), i) for i in 1:n_basefuncs]
+        # @testset let x = sample_random_point(ref_shape) # not compatible with Julia 1.6
+        @testset "Random point test" begin
+            x = sample_random_point(ref_shape)
+            # Check gradient evaluation
+            @test vec(ForwardDiff.jacobian(f, Array(x))') ≈
+                reinterpret(Float64, [shape_gradient(interpolation, x, i) for i in 1:n_basefuncs])
+            # Check partition of unity at random point.
+            ansatz_sum = sum([shape_value(interpolation, x, i) for i in 1:n_basefuncs])
+            if ansatz_sum ≉ 1.0
+                # Show coordinate in case failure (see issue #811)
+                # Remove after 1.6 is removed from CI (see above)
+                println("Partition of unity test might fail at $x for $interpolation !")
+                @test ansatz_sum ≈ 1.0
+            end
```

Comment:
Not necessary, but learned this on Slack today, perhaps a nicer way to print if it fails?
(Haven't checked julia version compat for this)
```suggestion
            ansatz_test = @test sum([shape_value(interpolation, x, i) for i in 1:n_basefuncs]) ≈ 1.0
            if ansatz_test isa Test.Fail
                # Show coordinate in case failure (see issue #811)
                # Remove after 1.6 is removed from CI (see above)
                println("Partition of unity test failed at $x for $interpolation !")
            end
```

Reply:
Maybe better like this? So we have also context if another test in this battery fails.

Reply:
Yes! But looking forward to skipping 1.6 from CI:)